src/services/compare/handlers/sendDoesNotExistAlert.js

```diff
@@ -33,25 +40,115 @@ exports.handler = async function (event, context, callback) {
         message: `Alert for ${data.id} - TEST `,
       });
     } else {
-      const { emailRecipients, sourceEmail } = await getSecretsValue({
+      let isProgramTypeChp = false;
+      if (data.programType == "CHP") {
+        secretId = `${project}/${stage}/alerts/CHP`;
+        isProgramTypeChp = true;
+      }
+      //{ emailRecipients, sourceEmail, emailRecipientsA, emailRecipientsB }
+      const {
+        emailRecipients,
+        sourceEmail,
+        emailRecipientsA,
+        emailRecipientsB,
+      } = await getSecretsValue({
         region,
         secretId,
       });
+      let recipientType;
+      let recipients;
+      // if it greater then 2 days but less then 4 days
+      if (
+        data.secSinceMmdlSigned > 48 * 3600 &&
+        data.secSinceMmdlSigned < 48 * 2 * 3600
+      ) {
+        recipientType = "emailRecipientsA";
+        recipients = emailRecipientsA;
+        console.table({
+          secSinceMmdlSigned: data.secSinceMmdlSigned,
+          twoDays: 48 * 3600,
+          recipients,
+          recipientType,
+          "(data.secSinceMmdlSigned > (48 * 3600)) && (data.secSinceMmdlSigned < ((48 * 2) * 3600))":
+            data.secSinceMmdlSigned > 48 * 3600 &&
+            data.secSinceMmdlSigned < 48 * 2 * 3600,
+        });
+      }
+      // if it is greater then 4 days
+      else if (data.secSinceMmdlSigned > 48 * 2 * 3600) {
+        recipientType = "emailRecipientsB";
+        recipients = emailRecipientsB;
+        console.table({
+          secSinceMmdlSigned: data.secSinceMmdlSigned,
+          twoDays: 48 * 3600,
+          recipients,
+          recipientType,
+          "(data.secSinceMmdlSigned > (48 * 3600))":
+            data.secSinceMmdlSigned > 48 * 3600,
+        });
+      }
+      // if it is less then 2 days
+      else {
+        recipientType = "emailRecipients";
+        recipients = emailRecipients;
+        console.table({
+          secSinceMmdlSigned: data.secSinceMmdlSigned,
+          twoDays: 48 * 3600,
+          recipients,
+          recipientType,
+          "(data.secSinceMmdlSigned < (48 * 3600))":
+            data.secSinceMmdlSigned < 48 * 3600,
+        });
+      }
```

Comment:
there has to be a better way of doing this. instead of putting the recipients into different groups in the secrets, lets just give each of them a number of seconds which we can compare against to see if they should be included in the recipient list. 

i picture it as something like this: 

// seconds in a day: 86400
// seconds in two days: 172800
// seconds in 4 days: 345600

const isCHP = data.programType === "CHP"

// the secret would look something like this
const secrets = {
mmdl: {
chp: [
	{email: "someone@example.com", alertIfGreaterThanDays: 172800},
  {email: "someoneElse@example.com", alertIfGreaterThanSeconds: 345600 }
  ],
nonchp: [
	{email: "someone@example.com", alertIfGreaterThanSeconds: 172800},
  {email: "someoneElse@example.com", alertIfGreaterThanDays: 345600}
  ]
},
appian: []
}

// and the logic for if a secret does exist would go something like this: 

let recipients, template
if (data.secSinceMmdlSigned <= 172800) {
recipients = 
	secrets.mmdl[isCHP : "chp": "nonchp"]
	.filter(r => r.alertIfGreaterThanDays <= 172800)
  .map(r => r.email)
template = SendNoMatchTemplateInitial
} else {
 // .... same kinda logic here for diff template, and recipients
}

lmk if you want some to take a look at it together